### PR TITLE
Support moreh arange

### PR DESCRIPTION
**Pull request by hschoi4448**
support moreh arange

### CODE REVIEW

File: tt_eager/tt_lib/csrc/operations/primary/module.hpp [hunk 1]
```diff
@@ -496,6 +497,25 @@ void py_module(py::module& m_primary) {
...
+
+    m_primary.def(
+        "moreh_arange",
+        &moreh_arange,
+        py::arg("start"),
+        py::arg("end"),
+        py::arg("step"),
+        py::arg("any").noconvert(),
+        py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
+        "Performs a arange operation. Returns a output tensor.");
```
Comment (dongjin-na) [inline tt_eager/tt_lib/csrc/operations/primary/module.hpp:509]:
```suggestion
        "Performs an arange operation. Returns an output tensor.");
```

File: tt_eager/tt_lib/csrc/operations/primary/module.hpp [hunk 2]
```diff
@@ -496,6 +497,25 @@ void py_module(py::module& m_primary) {
...
+
+    m_primary.def(
+        "moreh_arange_inplace",
+        &moreh_arange_inplace,
+        py::arg("input_tensor").noconvert(),
+        py::arg("start"),
+        py::arg("end"),
+        py::arg("step"),
+        "Performs a arange operation. Returns a output tensor.");
```
Comment (dongjin-na) [inline tt_eager/tt_lib/csrc/operations/primary/module.hpp:518]: This is the same as the suggestion above.

File: tt_eager/tt_lib/csrc/operations/primary/module.hpp (hunk 1 above)
Comment (hschoi4448) [reply tt_eager/tt_lib/csrc/operations/primary/module.hpp]: I updated to "Performs an arange operation. Returns an output tensor."

File: tt_eager/tt_lib/csrc/operations/primary/module.hpp (hunk 2 above)
Comment (hschoi4448) [reply tt_eager/tt_lib/csrc/operations/primary/module.hpp]: I updated to "Performs an arange operation. Returns an output tensor."

**Review by dongjin-na — APPROVED** [review]
LGTM. I think it would be good to support row-major layout tiles later.

**Comment by hschoi4448** [comment]
close this pr, because can't solve conflicts

**Pull request closed without merging**